site/content/vendir/docs/develop/vendir-spec.md

```diff
@@ -116,6 +116,8 @@ directories:
         name: my-image-auth
       # specify wether to skip TLS verification; defaults to false (optional;v0.18.0+)
       dangerousSkipTLSVerify: false
+      # specify the response timeout in seconds for imgpkg when querying the registry; defaults to 30 (optional; v0.40.0+)
+      dangerousSkipTLSVerify: 30
```

Comment:
Is this the correct variable name?

Reply:
Variable should be `ResponseHeaderTimeout` ?

Reply:
yeah that's embarrassing. Sorry about that and it's updated now!